### PR TITLE
removed brew cask install android studio instruction

### LINKED ISSUE #6722: Update CONTRIBUTING_MACOS.md to reflect latest Android buildchain
Current version of `CONTRIBUTING_MACOS.md` lists `brew cask install android-studio` as the preferred way to install Android Studio, which is outdated (v1.4 vs v2.2.1). Let's review the document and update it accordingly.

/cc: @cammace

### PR DESCRIPTION
**Pull request by cammace**
Closes #6722

I agree that the most common/easy way to install Android Studio would be simply going to their website. I made this clear in the instructions by removing the `brew` command and instead suggesting to download from site.

cc: @zugaldia for review

### CODE REVIEW

**Comment by zugaldia** [comment]
> Note that you'll still need to `brew install java` for it to work.

I don't use `brew` to install Java either. I find it easier to simply to grab it from [Oracle](http://www.oracle.com/technetwork/java/javase/downloads/index.html)'s website. Also, I've been using Java 8 lately (forcing `sourceCompatibility = 1.7` and `targetCompatibility = 1.7`) and found no problems so far. What's @mapbox/android doing these days?

**Comment by cammace** [comment]
I've changed the instruction to link to Oracles download page were you can download the latest JDK version.

File: platform/android/CONTRIBUTING_MACOS.md [hunk 1]
```diff
@@ -2,14 +2,7 @@
...
 
-    brew tap caskroom/cask
-    brew install brew-cask
-    brew cask install java
-
-    brew cask install android-studio
-
-You can [download Android Studio instead of installing it with Homebrew, but you'll still need to `brew install java`](https://developer.android.com/sdk/index.html)
-for it to work.
+Head over to [Oracles download page](http://www.oracle.com/technetwork/java/javase/downloads/index.html) to get the Java Development Kit (JDK). Once JDK is installed and setup, you can grab the [latest version of Android Studio here](https://developer.android.com/sdk/index.html) and install it following the on screen instructions.
```
Comment (zugaldia) [inline platform/android/CONTRIBUTING_MACOS.md]: Nit, `Oracle's` and `set up`.

Comment (cammace) [inline platform/android/CONTRIBUTING_MACOS.md]: 👍 Made the changes.

**Pull request merged**